keras_nlp/models/bert/bert_preprocessing.py

```diff
@@ -63,7 +63,8 @@ class BertPreprocessor(keras.layers.Layer):
 
     Examples:
     ```python
-    vocab = ["[UNK]", "the", "qu", "##ick", "br", "##own", "fox", "tripped"]
+    vocab = ["[PAD]", "[UNK]", "[CLS]", "[SEP]",
```

Comment:
Aren't you missing a right paren on line 66?

Reply:
oops forgot to push latest, fixed